utils/setup.sh

```diff
@@ -1,17 +1,52 @@
 #!/bin/bash
 
-# Variables to modify to fit current environment specifications
-domain=INSERT_DOMAIN
-cloudfront_domain=INSERT_CLOUDFRONT_DOMAIN
-keystore_password=INSERT_KEYSTORE_PASSWORD
-redirect_location=google.com
-#####
-
-# DO NOT modify the following variables
-keystore="${domain}.store"
-keystore_path="/tools/Malleable-C2-Profiles/normal/${keystore}"
-c2_profile="${domain}-$(date '+%Y-%m-%d').profile"
-######
+function usage {
+  echo -e "Megazord setup script\n"
+  echo "This script will perform setup tasks to initialize the environment\
+prior to launching megazord."
+  echo -e "\nThese tasks include:"
+  echo "    - Setting values in the .env file used by docker compose"
+  echo "    - Generating a pseudorandom string for the uploads directory"
+  echo "    - Extracting certificates and private key from the keystore"
+  echo "    - Generating a fresh C2 Profile and outputting it in the correct location"
+  echo "    - Generating an htaccess config file for Apache and outputing it in the correct location"
+  echo "    - Generating a fresh Corefile and outputting it in the correct location"
+  echo
+  echo "Options:"
+  echo "    -c    The cloudfront url"
+  echo "    -d    The domain of the teamserver"
+  echo "    -p    The password for the teamserver"
+  echo "    -r    The url Apache will redirect out-of-scope traffic to"
+  echo
+  echo "Usage:"
+  echo "    ./setup.sh -r redirect-url.com -d c2domain.com -c cloudfront.domain.net"
```

Comment:
I always prefer this for Bash scripts as opposed to hard-coded values.

```suggestion
  echo "    ${0##*/} -r redirect-url.com -d c2domain.com -c cloudfront.domain.net"
```